guide-build-connectors.md

```diff
@@ -0,0 +1,36 @@
+1. Identify asset data and the relationships between those identifiers.  CAR-connector-schema.xls file can be helpful to figure out the mapping between CAR data model and source data structure.
```

Comment:
This should link out to the schema repo. We can't assume someone looking at this document even knows what the schema is.